megamek/src/megamek/common/verifier/TestSmallCraft.java

```diff
@@ -436,9 +446,15 @@ public double getWeightAmmo() {
     @Override
     public double getWeightMisc() {
         double weight = 0.0;
+        boolean NC3 = hasNC3(smallCraft);
         // Add in extra fire control system weight for exceeding base slot limit
         for (double extra : extraSlotCost(smallCraft)) {
-            weight += extra;
+            //Naval C3 doubles the weight of FCS, if present
+            if (NC3 == true) {
```

Comment:
This is already a boolean value and there is no reason to add an extra comparison operation. Simply `if (NC3)`